examples/ingest/dropbox/ingest.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# Processes several files in a nested folder structure from dropbox://utic-test-ingest-fixtures/ 
+# through Unstructured's library in 2 processes.
+# Due to Dropbox's interesting sdk, to list and get the root folders/files we use "dropbox:// /"
```

Comment:
do they have a page that describes this pattern (that we can link to)?

what does getting a nested path look like? `dropbox:// /some/other/path`?
is there ever a time when we don't have the `dropbox://` prefix or when that turns into a longer page `dropbox://something/else`?

Reply:
Yes. If we are not going from the root folder (the app folder) we can just use a simple path

`dropbox://nested-1`

Reply:
The whole root as " " really put a wrench into the simplicity of this thing.

Reply:
I will rewrite the comment to make more sense.

Reply:
Ahh, okay, got it. Totally makes sense. I mean objectively confusing on their side, but am tracking now. Thanks!